superset/assets/javascripts/components/FilterableTable/FilterableTable.jsx

```diff
@@ -0,0 +1,192 @@
+import { List } from 'immutable';
+import React, { PropTypes, PureComponent } from 'react';
+import {
+  Column,
+  Table,
+  SortDirection,
+  SortIndicator,
+} from 'react-virtualized';
+import { getTextWidth } from '../../modules/visUtils';
+
+require('./FilterableTableStyles.css');
```

Comment:
we've got the css loader setup in webpack, and this is the same way we require the css in other parts of sql-lab
https://github.com/airbnb/superset/blob/master/superset/assets/javascripts/SqlLab/index.jsx#L17